#! /bin/sh -e
## ubuntu-disable-browsing.dpatch by  <martin.pitt@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable Browsing by default on Ubuntu (not sanctioned yet by
## DP: https://wiki.ubuntu.com/DefaultNetworkServices). This is only applied
## DP: when building on Ubuntu.

[ "`lsb_release -is 2>/dev/null`" = "Ubuntu" ] || { echo -n '(ignored)' >&2; exit 0; }
. /usr/share/dpatch/dpatch-run

@DPATCH@
diff -urNad cups-1.3.0~rc2~/conf/cupsd.conf.in cups-1.3.0~rc2/conf/cupsd.conf.in
--- cups-1.3.0~rc2~/conf/cupsd.conf.in	2007-07-25 01:40:03.000000000 +0100
+++ cups-1.3.0~rc2/conf/cupsd.conf.in	2007-08-01 15:27:56.000000000 +0100
@@ -19,9 +19,10 @@
 @CUPS_LISTEN_DOMAINSOCKET@
 
 # Show shared printers on the local network.
-Browsing On
+Browsing Off
 BrowseOrder allow,deny
 BrowseAllow all
+BrowseAddress @LOCAL
 
 # Default authentication type, when authentication is required...
 DefaultAuthType Basic
